src/overlays/actors/ovl_Obj_Mu_Pict/z_obj_mu_pict.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_obj_mu_pict.c
  * Overlay: ovl_Obj_Mu_Pict
- * Description:
+ * Description: Garo/Gibdo poster in the Music Box House
```

Comment:
Looking at the code, this looks more like it's Pamela's father's interactions with you when you inspect his house after he's cured, not sure what to call that, but its not strictly the gibdo poster, its a more like a complicated tatl talking spot.

It doesn't draw the poster either, its draw function is empty, that's part of the scene mesh I think

Reply:
That's true, but the actors are clearly logically tied to the posters. You can see this easily in kz:
![NKZE01_2021-11-24_17-32-19](https://user-images.githubusercontent.com/12245827/143346889-94c74c06-4310-4db3-b63d-1ffcd5f211b7.png)
![NKZE01_2021-11-24_17-32-29](https://user-images.githubusercontent.com/12245827/143346966-e04bb1c6-7626-4723-8b4c-cfb6d1eac610.png)

I'll update the description to "Handles dialogue for checking the Garo/Gibdo poster in the Music Box House", since that makes it a little more clear that it doesn't actually draw the posters themselves.